### PR TITLE
Add new case of updating iface

### PR DESCRIPTION
**Pull request by chloerh**
- update-device for interface defined with not supported live-update
configurations

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Review by yalzhang — APPROVED** [review]

File: libvirt/tests/src/virtual_network/iface_update.py [hunk 1]
```diff
@@ -92,6 +96,21 @@ def run(test, params, env):
...
+            disk_boot = params.get('disk_book', 1)
+            vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
+            # Remove os boot config
+            vm_os = vmxml.os
+            vm_os.del_boots()
+            vmxml.os = vm_os
+            # Add boot config to disk
+            disk = vmxml.get_devices('disk')[0]
+            target_dev = disk.target.get('dev', '')
+            logging.debug(target_dev)
```
Comment (kylazhang) [inline libvirt/tests/src/virtual_network/iface_update.py:110]: Better give more readable message for the debug info

**Review by kylazhang — COMMENTED** [review]
Others are good for me

File: libvirt/tests/src/virtual_network/iface_update.py (hunk 1 above)
Comment (chloerh) [reply libvirt/tests/src/virtual_network/iface_update.py:110]: Of course, udpateing

Comment (chloerh) [reply libvirt/tests/src/virtual_network/iface_update.py]: Updated.

**Review by kylazhang — APPROVED** [review]

**Pull request merged**